### PR TITLE
upgrade to containerizers section

### PR DESCRIPTION
**Pull request by sascala**
## Urgency
- [ ] Blocker 
- [X] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: 1.8/usage/containerizers/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+nav_title: Using Containerizers
+menu_order: 40
+---
+
+Containerizers allow you to run tasks in containers. Running tasks in containers offers a number of benefits, including the ability to isolate tasks from one another and control task resources programmatically.
+
+DC/OS supports the Mesos containerizer types :
```
Comment (joel-hamill) [inline 1.8/usage/containerizers/index.md]: extra space after `types`

File: 1.8/usage/containerizers/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- The [original Mesos containerizer](/docs/1.8/usage/containerizers/mesos-containerizer/).
+
+- The [DC/OS Universal Container Runtime](/docs/1.8/usage/containerizers/ucr/)
```
Comment (joel-hamill) [inline 1.8/usage/containerizers/index.md]: missing period

**Review by joel-hamill — CHANGES_REQUESTED** [review]
You must have `nav_title`, but `post_title` is optional.  Some of the files are missing `post-title`.

**Comment by sascala** [comment]
@joel-hamill I think I got all your comments

**Review by joel-hamill — APPROVED** [review]

**Pull request merged**